### PR TITLE
Fix flags paths for OSX

### PR DESCRIPTION
**Pull request by yannvery**
This allows the installation of the lib on OSX system using brew to manage dependencies.

### CODE REVIEW

**Comment by gearnode** [comment]
@galdor that fix issue when macOS user use brew instead of macports.

**Comment by gearnode** [comment]
@yannvery This work without other setup with brew?

File: GNUmakefile [hunk 1]
```diff
@@ -30,9 +30,12 @@ endif
 ifeq ($(PLATFORM), Darwin)
 	CFLAGS += -I/opt/local/lib/erlang/usr/include
 	CFLAGS += -I/opt/local/include
+	CFLAGS += -I/usr/local/opt/erlang/usr/include
+	CFLAGS += -I/usr/local/opt/icu4c/include
```
Comment (galdor) [inline GNUmakefile:34]: This one should absolutely be added by `icu-config --cflags`. Could you give me the output ?
Any way could you add a small comment right to each include path, to understand the context ? From what I understand, `/opt/local/lib/erlang/usr/include` and `/opt/local/include` are for macports, and the others for brew.

Comment (yannvery) [reply GNUmakefile:34]: `icu-config --cflags` returns nothing

Concerning the paths, Macport uses:

- /opt/local/lib/erlang/usr/include
- /opt/local/include

and Brew:

- /usr/local/opt/erlang/usr/include
- /usr/local/opt/icu4c/include

Comment (ghost) [reply GNUmakefile:34]: Ok then I can tell you that there is something really wrong with the Brew ICU package.  There's not much I can do, but maybe some good soul could send them a PR to fix this mess.

**Pull request merged**